cityguide/source/app/src/main/java/org/kaaproject/kaa/demo/cityguide/fragment/PlacesFragment.java

```diff
@@ -119,7 +119,7 @@ public void onEvent(Events.ConfigurationUpdated configurationUpdated) {
 
     @Override
     public String getTitle() {
-        return mAreaName + "_" + mCityName;
```

Comment:
But with Netherlands_Amsterdam user understand where he is in this app. It's too important for user navigation